pkg/cli/iam_handle.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/abcxyz/access-on-demand/apis/v1alpha1"
+	"github.com/abcxyz/access-on-demand/pkg/handler"
+	"github.com/abcxyz/pkg/cli"
+	"gopkg.in/yaml.v3"
+
+	resourcemanager "cloud.google.com/go/resourcemanager/apiv3"
+)
+
+var _ cli.Command = (*IAMHandleCommand)(nil)
+
+type IAMHandleCommand struct {
+	cli.BaseCommand
+
+	flagPath string
+
+	flagDuration time.Duration
+
+	handler *handler.IAMHandler
```

Comment:
I'd prefer this to be an interface so this package will be decoupled from `IAMHandler`

Reply:
hum.....ok, this will make the test easier then just implement a fake Do. let me try it...